CHANGELOG.txt

```diff
@@ -14,11 +14,6 @@ RENAMED - A function/class has been renamed.
 REMOVED - A function/class has been removed.
 ======================================================================================================
 
-64.2.2 - 2022-03-08
```

Comment:
This is an artifact of us moving from SVN to Github, and will likely need to be done for many of the PRs, but this should be updated with the CHANGELOG.txt in the master branch